docs/actions.rst

```diff
@@ -2,16 +2,349 @@
+     - If the module fails execution, the number of times to retry it. Integer value, default 0
+     - No
+   * - delay
+     - The retry interval, an integer value
+     - No
+   * - verbosity
+     - Verbosity level when running the module, a value between 1-4
+     - No
+
+run_job_template
+************
+
+Run a job template.
+
+.. note::
+    ``--controller_url`` and ``--controller_token`` cmd options must be provided to use this action
+
+.. list-table::
+   :widths: 25 150 10
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Required
+   * - name
+     - The name of the job template
+     - Yes
+   * - organization
+     - The name of the organization
+     - Yes
+   * - set_facts
+     - The artifacts from the playbook execution are inserted back into the rule set as facts
+     - No
+   * - post_events
+     - The artifacts from the playbook execution are inserted back into the rule set as events
+     - No
+   * - ruleset
+     - The name of the ruleset to post the event or assert the fact to, default is current rule set.
+     - No
+   * - retry
+     - If the playbook fails execution, retry it once, boolean value true|false
+     - No
+   * - retries
+     - If the playbook fails execution, the number of times to retry it. An integer value
+     - No
+   * - delay
+     - The retry interval, an integer value specified in seconds
+     - No
+   * - var_root
+     - If the event is a deeply nested dictionary, the var_root can specify the key name whose value should replace the matching event value. The var_root can take a dictionary to account for data when we have multiple matching events.
+     - No
+   * - job_args
+     - Additional arguments sent to the job template launch API. Any answers to the survey and other extra vars should be set in nested key extra_vars. Event(s) and fact(s) will be automatically included in extra_vars too.
+     - No
+
+post_event
+**********
+.. list-table::  Post an event to a running rule set in the rules engine
+   :widths: 25 150 10
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Required
+   * - event
+     - The event dictionary to post
+     - Yes
+   * - ruleset
+     - The name of the rule set to post the event, default is the current rule set name
+     - No
+
+Example
+
+.. code-block:: yaml
+
+      action:
+        post_event:
+          ruleset: Test rules4
+          event:
+            j: 4
+
+Example, using data saved with assignment
+
+.. code-block:: yaml
+
+      name: multiple conditions
+      condition:
+        all:
+          - events.first << event.i == 0
+          - events.second << event.i == 1
+          - events.third << event.i == events.first.i + 2
+      action:
+        post_event:
+          ruleset: Test rules4
+          event:
+            data: "{{events.third}}"
+
+
+| The events and facts prefixes have rule scope and cannot be accessed outside of
+| rules. Please note the use of Jinja substitution when accessing the event results.
```

Comment:
```suggestion
| The events and facts prefixes have rule scope and cannot be accessed outside of rules. Please note the use of Jinja substitution when accessing the event results.
```

Reply:
suggestion rejected, many things are in the same way, out of the scope of this PR